tools/buildtools/packmol/packmol.xml

```diff
@@ -0,0 +1,191 @@
+            <param name="packmol_inputs_0|input" value="water.pdb" ftype="pdb"/>
+            <param name="packmol_inputs_1|input" value="urea.pdb" ftype="pdb"/>
+            <param name="packmol_inputs_0|structureparams|number" value="1000"/>
+            <param name="packmol_inputs_1|structureparams|number" value="400"/>
+            <param name="packmol_inputs_0|structureparams|constraints_repeat_0|constraints|constraint" value="box"/>
+            <param name="packmol_inputs_0|structureparams|constraints_repeat_0|constraints|xmax" value="40.0"/>
+            <param name="packmol_inputs_0|structureparams|constraints_repeat_0|constraints|ymax" value="40.0"/>
+            <param name="packmol_inputs_0|structureparams|constraints_repeat_0|constraints|zmax" value="40.0"/>
+            <param name="packmol_inputs_1|structureparams|constraints_repeat_0|constraints|constraint" value="box"/>
+            <param name="packmol_inputs_1|structureparams|constraints_repeat_0|constraints|xmax" value="40.0"/>
+            <param name="packmol_inputs_1|structureparams|constraints_repeat_0|constraints|ymax" value="40.0"/>
+            <param name="packmol_inputs_1|structureparams|constraints_repeat_0|constraints|zmax" value="40.0"/>
+            <param name="allparams.seed" value="-1"/>
+            <output name="outfile" file="packmol_outfile2.pdb" compare="sim_size" delta="100000">
+                <assert_contents>
+                    <has_text text="HETATM 3000  O   HOH"/>
+                    <has_text text="HETATM 6200  H     2 B"/>
+                </assert_contents>
+            </output>
+            <output name="parameteroutfile" file="packmol_parameteroutfile_2.txt" compare="diff" lines_diff="6"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+.. class:: infomark
+
+**What it does**
+
+Packmol creates initial MD configurations from input molecules. For example, create a water and urea mixture.
+
+.. class:: infomark
+
+**How it works**
+
+- Select a molecule file type e.g. pdb,xyz
+- Select single molecule from your history (can select multiple)
+- For each molecule:
+   - choose how many molecules to create (number variable)
+   - optionally choose:
+       - molecular radius
+       - how to number the molecules (resnumber variable)
+   - choose one or more placement constraint(s). Each constraint has different parameters:
+       - for example, specify placement inside a cube of size 40 Angstroms placed at the origin
+       - all units are in Angstroms for distances and degrees for angles
+
+.. class:: infomark
+
+**Outputs created**
+
+- A pdb file, view this in Galaxy by clicking on visualise and choosing a viewer. The NGLViewer works well for small molecules.
+- A packmol input script (for debugging and repeatability)
+
+.. class:: infomark
+
+**User guide and documentation**
+
+- Packmol `userguide`_
+- Calculating the number of molecules using the `volume guesser`_
+
+.. class:: infomark
+
+**Features not included**
```

Comment:
Should we move this into a github issue or a readme? I'm not sure we should expose this to a user.

Reply:
https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/issues/8

Reply:
agreed, added a github issue.